lib/conman.rb

```diff
@@ -2,12 +2,17 @@
 
 class Conman
 
-  def initialize(ui)
+  def initialize(ui, contact_list)
     @ui           = ui
-    @contact_list = []
+    @contact_list = contact_list
   end
 
   def run
+    add_contacts
+    search_contacts
```

Comment:
That's where the menu will go :)
For now it's a placeholder (this PR was supposed to be just for the search but I got carried away :-P )